src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -19,17 +20,26 @@ interface CreateProjectFormViewProps {
   onSubmit: (e: any) => void;
 }
 
+const NERInput = styled(TextField)(({ theme }) => ({
```

Comment:
can we get this in cr create as well

Reply:
i mean the abstraction so it's not rewritten a million times

Reply:
yeah I'll take a look, I think I just started on this one and then was like fuck there's a bunch of places this needs to be added

Reply:
oh nvm, I see the issue, it's because createchange request uses the ReactHookTextField. I need to figure out how to make the border thing work consistently with that, it fucks up when the size of the input changes. For now, I can abstract out that code into a style variable